django_project/settings.py

```diff
@@ -107,6 +110,11 @@
 
 LANGUAGE_CODE = "en-us"
 
+LANGUAGES = [
+    ('en-us', _('English')),
```

Comment:
Why do we use  `ugettext_lazy` somewhere and `ugettext` elsewhere? What is the difference between those two?

Reply:
Makes sense